components/signature/events.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Events
+page_title: Signature - Events
+description: Events in the Signature for Blazor.
+slug: signature-events
+tags: telerik,blazor,signature,events
+published: true
+position: 20
+---
+
+# Events
+
+This article describes the Blazor Signature events and provides a runnable example with sample event handler implementations.
+
+* [OnBlur](#onblur)
+* [OnChange](#onchange)
+* [ValueChanged](#valuechanged)
+
+## OnBlur
+
+The `OnBlur` event fires when the Signature loses focus. 
+
+## OnChange
+
+The `OnChange` event represents a user action - confirmation of the current value. It fires when the user presses `Enter`, or when the component loses focus.
+
+>tip The `OnChange` event is a custom event and does not interfere with bindings, so you can use it together with models and forms.
+
+## ValueChanged
+
+The `ValueChanged` event fires when signature is fully drawn.
+
+## Example
+
+>caption Handle the Blazor Signature Events
+
+````CSHTML
+<p>Last event: @EventLog</p>
+
+<TelerikSignature Value="@SignatureValue"
+                  ValueChanged="@ValueChangedHandler"
+                  OnBlur="@OnSignatureBlur"
+                  OnChange="@OnSignatureChange"
+                  Width="300px"
+                  Height="300px">
+</TelerikSignature>
+
+@code {
+    private string SignatureValue { get; set; }
+
+    private string EventLog { get; set; } = "...";
+
+    private void ValueChangedHandler(string value)
+    {
+        SignatureValue = value;
+
+        EventLog = $"ValueChanged event fired";
```

Comment:
Time will improve the UX when the same event fires multiple times.

```suggestion
        EventLog = $"ValueChanged event fired at {DateTime.Now.ToLongTimeString()}";
```

Reply:
The event fires once when the signature is fully drawn. Do we need the timestamp?

Reply:
Some signatures require multiple touches (mouse clicks). Or, if you press Enter twice, the page will not reveal two events.